### DISCUSSION TITLE
Not sure if this is a great idea but lets try it

### DISCUSSION THREAD

**Discussion by rbygrave** [ebean-orm/ebean#2334] [General]
Hi, I am not sure if it is a great idea to use github dicussions but we could try it out and see how it goes.

Cheers, Rob.

**Comment by vladimirfx** [comment]
IMO it is a more convenient way than Google Groups, especially when concrete code is disscussed.

**Reply by rbygrave** [reply]
Cool.  This is the first project that I am using it but it's all good so far.  Might also be easier to search than google groups, not sure.  Yes, happy to continue using it and see if more people adopt using it.
